etna/transforms/encoders/mean_encoder.py

```diff
@@ -0,0 +1,309 @@
+import reprlib
+from enum import Enum
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Union
+
+import numpy as np
+import pandas as pd
+from bottleneck import nanmean
+from pandas import Timestamp
```

Comment:
I don't see any good reason for this import.

Reply:
I don't understand

Reply:
Why do we need this import? Can't we just use `pd.Timestamp`?

Reply:
Fixed